rust/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "libzcashlc"
+version = "0.0.8"
+authors = ["Jack Grigg <jack@z.cash>",
+            "Francisco Gindre <francisco@z.cash>",
+]
+edition = "2018"
+build = "build.rs"
+
+[dependencies]
+failure = "0.1"
+ffi_helpers = "0.2"
+hex = "0.4"
+zcash_client_backend = "0.5"
+zcash_client_sqlite = "0.3"
+zcash_primitives = "0.5"
+zcash_proofs = "0.5"
+
+#### Temporary additions: ####################################
+#bitvec = "0.22"
+base58 = "0.1.0"
+#sha2 = "0.9"
+#bs58 = { version = "0.3", features = ["check"] }
+#hdwallet = "0.3.0"
+#ripemd160 = "0.9"
+hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "72f1f7a56c114eed484cefd6d402b7ef28158712"}
+secp256k1 = "0.19"
+time = "0.2"
+num-bigint = "0.3.3"
+##############################################################
+
+#[dependencies.zcash_proofs]
+#version = "0.5"
+#default-features = false
+#features = ["local-prover"]
+
+[build-dependencies]
+cbindgen = "0.14"
+
+[lib]
+name = "zcashlc"
+path = "src/lib.rs"
+crate-type = ["staticlib"]
+
+[profile.release]
+lto = true
+
+[patch.crates-io]
+zcash_client_backend = {git = "https://github.com/gmale/librustzcash", branch = "autoshield-poc-daa-taddr3"}
```

Comment:
The latest code is patched to a different repo and branch. See https://github.com/zcash/ZcashLightClientKit/blob/0.12.0-beta.6/Cargo.toml

````Toml
[patch.crates-io]
zcash_client_backend = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_client_sqlite = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_primitives = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
zcash_proofs = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
````

Reply:
👍 will update

Reply:
👍 done in [1839606](https://github.com/zcash-hackworks/zcash-light-client-ffi/pull/1/commits/1839606a4b4219e9c10cb0df94d72765fbbf049f)